packages/webapp/src/components/forms/AddRequestForm/index.tsx

```diff
@@ -224,6 +224,13 @@ export const AddRequestForm: StandardFC<AddRequestFormProps> = wrap(function Add
 
 								<Row className='mb-4 pb-2'>
 									<Col>
+										<FormSectionTitle>
+											<>
+												{t('addRequestFields.description')}{' '}
```

Comment:
Why not placing your `{t('addRequestFields.description')}{' '}` inside a `<span>` to avoid using `<></>`?

Reply:
Here I am just following the same convention as the labels above